docs/.vuepress/theme/components/NavLink.vue

```diff
@@ -16,7 +16,22 @@
     :rel="rel"
     @focusout="focusoutAction"
   >
-    {{ item.text }}
+    <section v-if="item.subTexts?.length">
```

Comment:
```suggestion
    <section v-if="item.subTexts && item.subTexts.length">
```
With optional chaining, I've got errors in watch and build modes. I tried with fresh dependencies, and the results were the same.

```
client?cae4:159 ./.vuepress/theme/components/NavLink.vue?vue&type=template&id=0af88b9c& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"febc8548-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./.vuepress/theme/components/NavLink.vue?vue&type=template&id=0af88b9c&)
Module build failed (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
SyntaxError: Unexpected token (1:388)
    at Parser.pp$4.raise (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2757:13)
    at Parser.pp.unexpected (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:647:8)
    at Parser.pp$3.parseExprAtom (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2196:10)
    at Parser.<anonymous> (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:6003:24)
    at Parser.parseExprAtom (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:6129:31)
    at Parser.pp$3.parseExprSubscripts (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
    at Parser.pp$3.parseExprOps (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:1966:19)
    at Parser.pp$3.parseMaybeConditional (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:1949:19)
    at Parser.pp$3.parseMaybeAssign (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
    at Parser.pp$3.parseMaybeConditional (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:1954:28)
    at Parser.pp$3.parseMaybeAssign (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
    at Parser.pp$3.parseParenAndDistinguishExpression (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2238:30)
    at Parser.pp$3.parseExprAtom (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:2163:41)
    at Parser.<anonymous> (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:6003:24)
    at Parser.parseExprAtom (/Users/budnix/Documents/Projects/handsontable-develop/docs/node_modules/vue-template-es2015-compiler/buble.js:6129:31)
```

Reply:
It doesn't apply to my local env and staging, but I'll change it.